docs/releases/1.9.txt

```diff
@@ -313,6 +313,10 @@ Forms
   and trailing whitespace.  As this defaults to ``True`` this is different
   behavior from previous releases.
 
+* Form fields now support the :attr:`~django.forms.Field.disabled` argument,
+  allowing the field widget to be displayed disabled by browsers. The value is
```

Comment:
The second sentence might not be so clear. You might chop it or replace it with the one from the docs "Even if..."